.github/workflows/linux.yml

```diff
@@ -127,16 +127,19 @@ jobs:
         run: |
           meson test -C ../pgroonga.build -v
       - name: Show diff
-        if: failure()
+        # TODO: Remove "matrix.postgresql-unreleased == 'yes' ||" when we're PostgreSQL 18 ready
```

Comment:
In my opinion,  `matrix.postgresql-unreleased == 'yes'` does not need to remove.
Because I think we will use `matrix.postgresql-unreleased == 'yes'` for PostgreSQL 19 even after PostgreSQL 18 released.

Reply:
PGroongaでPostgreSQL 18がサポートできたら、この条件は消したい感じです。
PostgreSQL 18のリリース前でも消したい感じです。
リリース前でもサポートできたら他のバージョンと同様にテストを実行して失敗したときだけdiffを見たいからです。

ということで `unsupported` という条件ならば残しで良いと考えますが、 `unreleased` なので `ready` な状態になったら削除で良かろうと思います。